.travis.yml

```diff
@@ -96,7 +97,6 @@ deploy:
     keep-history: true
     on:
       branch: release
-      condition: $PUBLISH_DOC = true
```

Comment:
Since we are already restricting the deploy on the `release` branch, we don't need this ENV config anymore.